lib/src/main/res/values/attrs.xml

```diff
@@ -5,5 +5,6 @@
         <attr name="text" format="string" />
         <attr name="hint" format="string" />
         <attr name="removeDefaultStyles" format="boolean" />
+        <attr name="mask" format="string" />
```

Comment:
Is this definable through XML? Isn't it an array of either Regex or strings, not a string itself?

Reply:
Yes, if found the string is converted to a list during initialization.